test/modules/test_modules_find_updates.py

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+"""Tests for Sopel's ``find_updates`` plugin"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+import requests.exceptions
+
+from sopel.modules import find_updates
+
+
+TMP_CONFIG = """
+    [core]
+    owner = Admin
+    nick = Sopel
+    enable =
+        find_updates
+    host = chat.freenode.net
+    """
```

Comment:
```suggestion
[core]
owner = Admin
nick = Sopel
enable =
    find_updates
host = chat.freenode.net
"""
```

And then py2.7 will stop complaining.

Reply:
Aw hell, lol. I was so worried about py3.3 breaking that I didn't test 2.7 😹